decidim-system/db/migrate/20201120160614_fill_smtp_from_email.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class FillSmtpFromEmail < ActiveRecord::Migration[5.2]
+  def up
+    Decidim::Organization.all.each do |org|
+      if org.smtp_settings["from_email"].blank?
```

Comment:
Is `org.smtp_settings["from_email"]` working when there is a single organization where the `smtp_settings` have never been seted?
The case is when `org.smtp_settings` is nil

Reply:
Yes it is. To be precise, if `from_email` is empty, it returns a default dummy address (`change@example.org`).
The purpose of this migration is to correct previously unsetted from_email fields (in the case only label was defined as instance). However, as it is not mandatory, we have deleted this migration, and modify the CHANGELOG in order to inform developers about it.